bundle/Form/FieldTypeHandlerRegistry.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form;
+
+use OutOfBoundsException;
+use RuntimeException;
+use function gettype;
+use function is_callable;
+
+final class FieldTypeHandlerRegistry
+{
+    /**
+     * Map of registered callable or FieldTypeHandlerInterface objects.
+     */
+    private array $map;
+
+    /**
+     * Creates a service registry.
+     *
+     * In $map an array consisting of a mapping of FieldType identifiers to object / callable is expected.
+     * In case of callable factory FieldTypeHandlerInterface should be returned on execution.
+     */
+    public function __construct(array $map = [])
+    {
+        $this->map = $map;
+    }
+
+    /**
+     * Register a $service for FieldType $identifier.
+     *
+     * @param \Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandlerInterface|callable $handler
```

Comment:
I think we can remove `callable`, this was initially implemented for quite old SF version.

Reply:
solved in 706fed57